### ISSUE TITLE
TimeSpan: invalid week calculations

### ISSUE THREAD

**Issue by MykhailoAllakhverdov** [Humanizr/Humanizer#1009]
```
TimeSpan.FromDays(31).Humanize();
TimeSpan.FromDays(28).Humanize();
```

Both expressions return "4 weeks".

**Comment by AbdullahDibas** [comment]
@MykhailoAllakhverdov I don't think this is an issue, since in the first command where you call `TimeSpan.FromDays(31).Humanize();` the passed value for **precision** is 1 (the default value) based on the description of the Humanize method's **precision** parameter says:
_The maximum number of time units to return. Defaulted is 1 which means the largest unit is returned_
so if you call `TimeSpan.FromDays(31).Humanize(2);` then it will return different results : **4 weeks, 3 days**

**Comment by clairernovotny** [comment]
Thank you for the report. This is the documented `precision: 1` behavior: only the largest unit is returned. Passing `precision: 2` returns `4 weeks, 3 days` for 31 days, so there is no calculation bug to fix here.
